### PR TITLE
add credits API

### PR DESCRIPTION
**Pull request by lionelz**

### CODE REVIEW

File: alooma/alooma.py [hunk 1]
```diff
@@ -1551,6 +1551,41 @@ def publish_notification(self, level, description, data):
...
         return res
 
+    def get_used_credits(self, from_day=None, to_day=None, all_accounts=False):
+        """ Get the credits consumption per day for the asked period
+        for the whole company or for the login account
+        The current day used credits can change according to real use
+
+            :param from_day: 'YYYY-MM-DD' from date period, if None, no from limit
+            :param to_day: 'YYYY-MM-DD' to date history, if None, until today
+            :param all_accounts: if true, return the used credits for all accounts
```
Comment (yairwein) [inline alooma/alooma.py]: @lionelz getting for all accounts separates the result per account, yes?

File: alooma/alooma.py [hunk 2]
```diff
@@ -1551,6 +1551,41 @@ def publish_notification(self, level, description, data):
...
+        """
+        url = self.rest_url + "credits/used-credits"
```
Comment (yairwein) [inline alooma/alooma.py]: use string format?

File: alooma/alooma.py (hunk 1 above)
Comment (lionelz) [reply alooma/alooma.py]: The result is not separated, this is a list of credits with different instance names:
[{'date': '2018-07-02', 'instance_name': 'instance-name', 'value': 3.0}, ...]
The doc is updated to be clear.

File: alooma/alooma.py (hunk 2 above)
Comment (lionelz) [reply alooma/alooma.py]: datetime and string are supported in last commit.

**Pull request merged**